modules/nf-core/muscle/tests/main.nf.test

```diff
@@ -0,0 +1,98 @@
+nextflow_process {
+
+    name "Test Process MUSCLE"
+    config "./nextflow.config"
+    script "../main.nf"
+    process "MUSCLE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "muscle"
+
+    test("test_muscle") {
+
+        when {
+            params {
+                module_args = '-fasta -verbose -phys -phyi -maxiters 2'
+            }
+            process {
+                """
+                    input[0] = [[id:'test'],// meta map
+                        file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/alignment/informative_sites.fas', checkIfExists:true)]
+                """
+            }
+        }
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.aligned_fasta,
+                    process.out.phyi,
+                    process.out.phys,
+                    file(process.out.log[0][1]).name,
+                    process.out.versions
+                    ).match() }
+            )
+        }
+    }
+
+    test("test_muscle_maketree") {
+
+        setup {
+            run("MUSCLE", alias: "MUSCLE_SETUP") {
+                script "../main.nf"
+                params {
+                    module_args = '-fasta -verbose -phys -phyi -maxiters 2'
+                }
+
```

Comment:
As in `ultra/pipleine` I don't think this belongs here. 

You need to set these in the `when` block, probably naming them more like `muscle_setup_args` and then use that in the config. 

You will most likely need to set the args in the config using the alias name too as otherwise it will clash with the config for the actual `muscle` process under test

Reply:
I generally both of these using Usman's automatic conversion, maybe something needs looking at in that then

Reply:
Do you mean the `migrate-pytest` in tools?

Reply:
> Do you mean the migrate-pytest in tools?

No it's using `pytest2nf-test` we could effectively change how it's written in the tool.